uSync.BackOffice/SyncHandlers/SyncHandlerRoot.cs

```diff
@@ -1041,6 +1041,7 @@ public IEnumerable<uSyncAction> Export(Udi udi, string[] folders, HandlerSetting
         virtual public IEnumerable<uSyncAction> Export(TObject item, string folder, HandlerSettings config)
             => Export(item, [folder], config);
 
+        virtual public string GetDirectoryForItem(TObject item, string[] folders, HandlerSettings config) => folders.Last();
```

Comment:
I think we can get away with this being protected (not public) then we don't have breaking changes down the line when we add / remove a public method. 

That might also mean it doesn't need an XML comment for the build to work (but we should add one anyway?).

Reply:
Will add after bank holiday, protected should also work, will test it when I am back after long weekend :)

Reply:
Adjusted based on feedback :)